cellprofiler/library/functions/image_processing.py

```diff
@@ -17,3 +20,57 @@ def medial_axis(image):
     return skimage.morphology.medial_axis(image)
 
 
+def enhance_edges_sobel(image, mask=None, direction="all"):
+    if direction.casefold() == "all":
+        output_pixels = centrosome.filter.sobel(image, mask)
```

Comment:
Scipy has a Sobel filter, have we done any testing to compare results and/or performance?